robot_main_state_node:
  ros__parameters:
    waypoints_start_to_challenge_zone: 
      start: [0.0, 0.0, 0.0]
      sub: [1.75, 0.0, 0.0]
      challenge_zone: [1.75, 0.40, -90.0]

    lists_challenge_zone: 
      room1: [-0.098, 0.56, 178.5]
      room2: [0.344, 0.565, 180.0]
      room3: [0.818, 0.57, 180.0]
      room4: [1.303, 0.58, 180.0]
      room5: [1.775, 0.59, 180.0]